packages/ui/core-components/src/lib/atoms/inputs/slider/_Slider.svelte

```diff
@@ -163,6 +163,21 @@
 			}
 		}
 	}
+
+	// handle steps, slider lags when there are greater then 1000 steps/ticks between max and min
+	let sliderTicks = max - min;
+	let minStep = sliderTicks / 1000;
+	console.log(`minStep: ${minStep}`, name);
```

Comment:
```suggestion
```

Is this a debug log, or meant for the user?

Reply:
Nope, i'll remove